reference/5.1/Microsoft.PowerShell.Core/About/about_Using.md

```diff
@@ -153,32 +152,25 @@ $deck.Shuffle()
 
 ### Example 3 - Load classes from an assembly
 
-This example loads an assembly so that its classes can be used to create new
-PowerShell classes. The following script creates a new PowerShell class that is
-derived from **DirectoryContext** class.
+This example loads an assembly so that its classes can be used when processing
+data. The following script uses custom settings for serializing a hash table.
 
 ```powershell
-using assembly 'C:\Program Files\PowerShell\7\System.DirectoryServices.dll'
-using namespace System.DirectoryServices.ActiveDirectory
+using assembly 'C:\Program Files\PowerShell\7\Newtonsoft.Json.dll'
+using namespace Newtonsoft.Json
```

Comment:
> **Note**: Placed in draft until we address this

I used Newtonsoft here, but I think it might be more useful to instead specify [YamlDotNet][01] instead, since there's no built-in functionality for YAML and coming up with a coherent example for requiring custom JSON serialization eluded me.

[01]: https://github.com/aaubry/YamlDotNet

Reply:
Yeah, and Newtonsoft.Json gets loaded by PowerShell. We need an assembly that doesn't ship with PowerShell.